portfolio_ahardman_stylesheet.css

```diff
@@ -0,0 +1,339 @@
+	display: flex;
+	align-items: center;
+	background-image: linear-gradient(to right,rgb(110,110,110),rgb(180,180,180));
+}
+path{
+	stroke-width: 5px;
+	stroke: rgb(0, 204, 191);
+	fill: none;
+	stroke-linecap: round;
+}
+circle{
+	fill:rgb(0, 164, 153);
+}
+/* ----------------------navbar settings---------------------- */
+.navbar{
+	position:fixed;
+	position: sticky;
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	top: 0px;
+	height: 60px;
+}
+.navbar .link.container{
+	position: relative;
+	left:0;
+	z-index: 99999;
+	height: 52px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	background-color: rgb(180,180,180);
+	border: 5px solid rgb(0, 204, 191);
+	border-radius: 5px;
+}
+.navbar .link.container a{
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin: 1.8vw;
+	color: rgb(0,0,0);
+	font-size: 1.2rem;
+	transition-duration: 0.8s;
+}
+.navbar .link.container a:visited{
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	color: rgb(0,0,0);
+	text-decoration: none;
+}
+.navbar .link.container a:hover{
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	color: rgb(255,255,255);
+	transition-duration: 0.3s;
+}
+/* ----------------------main section settings---------------------- */
+main{
+	background-image: linear-gradient(to right,rgb(110,110,110),rgb(180,180,180));
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height:calc(100vh - 60px);
+}
+main .container{
+	display: flex;
+	justify-content: center;
+	flex-direction: column;
+	width: 70%;
+	margin:80px 0px 80px 0px;
+	padding:30px 0px 30px 0px;
+}
+main h1{
+	font-size: 3.2rem;
+	margin: 0px;
+	padding: 0px;
+	border: 0px;
+}
+main h3{
+	text-align: left;
+	font-size: 2rem;
+	margin: 2.5vh 0px 0px 0px;
+	padding: 0px;
+	border: 0px;
+}
+/* portfolio section */
+.portfolio{
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	background-color: white;
+}
+.portfolio .container{
+	width: 70%;
+	margin-top: 150px;
+	margin-bottom: 150px;
+}
+.portfolio h2{
+	display: flex;
+	align-items: center;
+	font-size: 2.5rem;
+	margin: 0px 0vh 60px 0vh;
+	padding: 0px;
+}
+.portfolio .container .builds_container{
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+	align-items: center;
+}
+.build{
+	margin: 10px;
+	padding-bottom: 10px;
+	display: flex;
+	justify-content: space-around;
+	align-items: flex-end;
+	font-size: 1rem;
+	width: 200px;
+	height: 200px;
+	transition-duration: 0.6s;
+	border: 3px solid rgb(0, 164, 153);
+	border-radius: 5px;
+}
+.popup{
+	padding: 10px;
+	background-color: rgb(0, 204, 191);
+	border: 1px solid black;
+	border-radius: 5px;
+	font-size: 1rem;
+}
+.build.box1{
+	background-image: url('https://raw.githubusercontent.com/Mayden-Academy/FST-curriculum/master/MAYDEN%20ACADEMY_WEBSAFE_LOGO_06.15.png?token=AORF4IAOEYU56RY3T3FACIK6M6EHG');
+	background-size: 100%;
```

Comment:
background size, repeat and position could just be put on `.build` so you dont have to repeat them